### DISCUSSION TITLE
测试表单提交发现无成绩，重试后发现是邮箱地址验证的时候，没有处理大小写

### DISCUSSION THREAD

**Discussion by jaxrunboo** [AmazingAng/WTF-Solidity#388] [General]
同上。我的邮箱地址在github显示的时候首字母大写，我也这样复制粘贴了，结果没有成绩。
然后我提交测试内容的时候，把邮箱首字母改成小写，成绩通过。
以上。

**Comment by AmazingAng** [comment]
> 同上。我的邮箱地址在github显示的时候首字母大写，我也这样复制粘贴了，结果没有成绩。 然后我提交测试内容的时候，把邮箱首字母改成小写，成绩通过。 以上。

感谢反馈，的确存在这个情况，我们会尽快修复这个bug
